forc-plugins/forc-client/src/op/deploy.rs

```diff
@@ -536,17 +438,16 @@ pub async fn deploy_executables(
     Ok(deployed_executable)
 }
 
-fn extract_data_offset(binary: &[u8]) -> Result<usize> {
-    if binary.len() < 16 {
+pub fn extract_configurables_offset(binary: &[u8]) -> Result<usize> {
+    if binary.len() < 24 {
```

Comment:
nit: magic number; maybe make the `24` const or assign var name.

Reply:
This is copied from the SDK since they do not export this module: https://github.com/FuelLabs/fuels-rs/blob/e2d874fc62ff97aa993ff25885987b47742f778c/packages/fuels-programs/src/assembly/script_and_predicate_loader.rs#L288-L290

Reply:
I added a code comment